Competition2018/src/main/java/competition/subsystems/elevator/commands/TrackCurrentUsageCommand.java

```diff
@@ -0,0 +1,61 @@
+package competition.subsystems.elevator.commands;
+
+import java.util.HashMap;
+
+import com.google.inject.Inject;
+
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.command.BaseCommand;
+import xbot.common.controls.actuators.XCANTalon;
+
+public class TrackCurrentUsageCommand extends BaseCommand {
+
+    XCANTalon talon;
+    double current;
```

Comment:
`current` and `time` don't need to be class-level; you can define them where you use them.

Because it never changes, `currentAveragingWindow` should be a `final int` and named in UPPER_SNAKE_CASE with its value assigned inline. Alternately, it could be a `DoubleProperty` so we can alter the window if desired.

Reply:
did it!